docs/docs/Configuration/configuration-backend-only.md

```diff
@@ -1,154 +1,120 @@
 ---
-title: Backend-Only
+title: Start Langflow in backend-only mode
 sidebar_position: 4
 slug: /configuration-backend-only
 ---
 
+Learn how to serve the [Basic Prompting flow](/starter-projects-basic-prompting) as an endpoint in backend-only mode.
 
+Langflow can run in `--backend-only` mode to expose a Langflow app as an API endpoint, without running the frontend UI.
+This is also known as "headless" mode. Running Langflow without the frontend is useful for automation, testing, and situations where you just need to serve a flow as a workload without creating a new flow in the UI.
 
-:::info
+To start Langflow in backend-only mode, pass the `--backend-only` flag at startup.
 
-This page may contain outdated information. It will be updated as soon as possible.
-
-:::
-
-
-
-
-You can run Langflow in `--backend-only` mode to expose your Langflow app as an API, without running the frontend UI.
-
-
-Start langflow in backend-only mode with `python3 -m langflow run --backend-only`.
+```python
+python3 -m langflow run --backend-only
+```
 
+The terminal prints `Welcome to ⛓ Langflow`, and a blank window opens at `http://127.0.0.1:7860/all`.
 
-The terminal prints `Welcome to ⛓ Langflow`, and a blank window opens at `http://127.0.0.1:7864/all`.
 Langflow will now serve requests to its API without the frontend running.
 
+## Prerequisites
 
-## Prerequisites {#81dfa9407ed648889081b9d08b0e5cfe}
+- [Langflow is installed](/getting-started-installation)
+- [You have an OpenAI API key](https://platform.openai.com/)
+- [You have a Langflow Basic Prompting flow](/starter-projects-basic-prompting)
 
-- [Langflow installed](/getting-started-installation)
-- [OpenAI API key](https://platform.openai.com/)
-- [A Langflow flow created](/starter-projects-basic-prompting)
+## Get your flow's ID
 
-## Download your flow's curl call {#d2cf1b694e4741eca07fd9806516007b}
+This guide assumes you have created a [Basic Prompting flow](/starter-projects-basic-prompting) or have another working flow available.
 
-1. Click API.
-2. Click **curl** &gt; **Copy code** and save the code to your local machine.
+1. In the Langflow UI, click **API**.
+2. Click **curl** &gt; **Copy code** and copy the code to your local machine.
```

Comment:
You wrote "and copy the code to your local machine". Since you already copied it by clicking "Copy code", wouldn't you be pasting the code to your local machine? And where on your local machine? Any requirements or recommendations for location on the local machine?